src/main/java/org/cryptomator/frontend/fuse/FuseNioAdapterComponent.java

```diff
@@ -23,6 +24,9 @@
 		@BindsInstance
 		Builder maxFileNameLength(@Named("maxFileNameLength") int maxFileNameLength);
 
+		@BindsInstance
+		Builder toFuseEncoder(@Named("toFuseEncoder") BufferEncoder toFuseEncoder);
```

Comment:
I don't understand the name `toFuseEncoder`. See my comment in #56 regarding naming the two different sides in a transcoding process.